### ISSUE TITLE
Feature Request: rest/api/sql/servers/list should include resource group for the sql server.

### ISSUE THREAD

**Issue by jwshive** [Azure/azure-rest-api-specs#13238]
Looking at this endpoint
[https://docs.microsoft.com/en-us/rest/api/sql/servers/list](https://docs.microsoft.com/en-us/rest/api/sql/servers/list)

The return of the list request should include the resource group as part of the response. It's in the ID section as part of the long id string, but that's not as easy to pull out as it would be it's own resource definition.

**Comment by ericshape** [comment]
@jwshive Thanks for your feedback!
we had list by resource group in the latest API version: https://docs.microsoft.com/en-us/rest/api/sql/2020-08-01-preview/servers/listbyresourcegroup

**Comment by jwshive** [comment]
Yes but that's not the ask and not my use case. I want to see all servers and not have to know their resource group to start with. I want to get that from a list of all the servers.
